src/omv/ports/stm32/modules/py_lcd.c

```diff
@@ -208,27 +212,27 @@ void spi_lcd_callback(SPI_HandleTypeDef *hspi)
         switch (spi_tx_cb_state) {
             case SPI_TX_CB_MEMORY_WRITE_CMD: {
                 if (!spi_tx_cb_state_on[framebuffer_head]) {
-                    spi_tx_cb_state = SPI_TX_CB_DISPLAY_OFF;
-                    framebuffer_tail = framebuffer_head;
                     OMV_SPI_LCD_CS_HIGH();
                     OMV_SPI_LCD_RS_ON();
+                    spi_tx_cb_state = SPI_TX_CB_DISPLAY_OFF;
+                    framebuffer_tail = framebuffer_head;
                     OMV_SPI_LCD_CS_LOW();
-                    HAL_SPI_Transmit_IT(OMV_SPI_LCD_CONTROLLER->spi, (uint8_t []) {0x28}, 1); // display off
+                    HAL_SPI_Transmit_IT(OMV_SPI_LCD_CONTROLLER->spi, (uint8_t *) display_off, sizeof(display_off));
```

Comment:
@kwagyeman Why not just use transmit, it's 1 byte ?

Reply:
It's an interrupt FSM. Transmit IT when finished generated a tx done interrupt moving the FSM to the next state. Also, that would block during the interrupt.

Reply:
Moving the CS line high code around adds a few nanoseconds of high time which fixes issues with the CS toggling too quickly.